src/Messages/IPointToInterlockingConnection/IPointToInterlockingConnection.cs

```diff
@@ -0,0 +1,33 @@
+using System.Threading.Tasks;
+
+namespace EulynxLive.Messages.IPointToInterlockingConnection;
+
+public interface IPointToInterlockingConnection: System.IDisposable {
+    Task SendPointPosition(PointState state);
+    Task SendTimeoutMessage();
+    void Connect();
+    Task<bool> InitializeConnection(PointState state);
+    public Task<PointPosition?> ReceivePointPosition();
+
+    public record PointState
+    {
+        public PointPosition PointPosition { get; set; }
+        public DegradedPointPosition DegradedPointPosition { get; set; }
+    }
+
+    public enum PointPosition
+    {
+        LEFT,
```

Comment:
this does not follow the .NET naming conventions for enumerations: https://learn.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions